src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/GitContentManager.java

```diff
@@ -115,7 +116,9 @@ public GitContentManager(final GitDb database, final ISearchProvider searchProvi
             log.info("API Configured to only allow published content to be returned.");
         }
 
-        this.cache = CacheBuilder.newBuilder().softValues().expireAfterAccess(1, TimeUnit.DAYS).build();
+        this.cache = CacheBuilder.newBuilder().recordStats().softValues().expireAfterAccess(1, TimeUnit.DAYS).build();
+        CACHE_METRICS_COLLECTOR.addCache("GitContentManagerCache", cache);
```

Comment:
For Prometheus metric name consistency:
```suggestion
        CACHE_METRICS_COLLECTOR.addCache("git_content_manager_cache", cache);
```

Reply:
Missed that, thanks.